packages/slider/src/Slider.ts

```diff
@@ -265,8 +275,8 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
                     @click=${this.handleLabelClick}
                     size=${this.size}
                 >
-                    ${this.slotHasContent ? nothing : this.label}
-                    <slot>${this.label}</slot>
+                    <span>${this.label}</span>
```

Comment:
we should render this is `this.slotHasContent`. Now with this implementation `this.label` will always be rendered?

Reply:
FYI A slider may or maynot have a label!

Reply:
updated it! 👍